queue/src/fiskaltrust.Middleware.Storage.MySQL/MySQLStorageConfiguration.cs

```diff
@@ -7,6 +7,9 @@ public class MySQLStorageConfiguration
     {
         [JsonProperty("connectionstring")]
         public string ConnectionString { get; set; }
+        
+        [JsonProperty("maxpoolsize")]
```

Comment:
You can leave ths is out because `JsonConvert.DeserializeObject` is case insensitive (IDK why we've set that for the connectionstring ^^)